plugins/woocommerce/changelog/improve-hpos-cli-verbose-output

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+use wp_json_encode instead of print_r in CLI output
```

Comment:
I wonder if this should be more specific?

```suggestion
Use wp_json_encode instead of print_r in output for HPOS-related WP CLI commands.
```

Reply:
@barryhughes That looks better. Committed the suggestion.